control/mpc_lateral_controller/src/mpc_lateral_controller.cpp

```diff
@@ -227,6 +231,28 @@ std::shared_ptr<SteeringOffsetEstimator> MpcLateralController::createSteerOffset
   return steering_offset_;
 }
 
+void MpcLateralController::setStatus(
+  diagnostic_updater::DiagnosticStatusWrapper & stat, const bool & m_is_mpc_solved)
```

Comment:
`m_is_mpc_solved` is defined as a member variable, so I don't think you need to declare it as an argument here.

Reply:
fixed in 74b41065e172bbff1e87bc9ccbe91af3be8d0eaf